src/main/java/org/mondemand/Client.java

```diff
@@ -71,6 +69,7 @@
   private ConcurrentHashMap<EventType, List<Transport>> transports = null;
   private ClientStatEmitter autoStatEmitter = null;
   private Thread emitterThread = null;
+  private Integer maxNumMetrics = null;
```

Comment:
would it not be better if we defined the default value for this here, and then overwrite it in the set..() method if we need to?